packages/blockchain-wallet-v4-frontend/src/components/Chart/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { defaultStyles, TooltipWithBounds as VisxTooltipWithBounds } from '@visx/tooltip'
+
+import { TooltipCard } from './styles'
+import { TooltipComponent } from './types'
+
+export const Tooltip: TooltipComponent = ({ children, left, offsetLeft, offsetTop, top }) => {
+  return (
+    <VisxTooltipWithBounds
```

Comment:
why can't this be just `TooltipWithBounds`?

Reply:
I added this wrapper so I could remove the shadow and padding that visx adds by default